tests/static-analysis/assert-isEmpty.php

```diff
@@ -4,15 +4,88 @@
 
 use Webmozart\Assert\Assert;
 
-
 /**
  * @psalm-pure
  *
  * @return null
  */
-function consume(?object $value)
+function isEmptyNullableObject(?object $value)
+{
+    Assert::isEmpty($value);
+
+    return $value;
+}
+
+/**
+ * @psalm-pure
+ *
+ * @return ""|"0"
```

Comment:
Tbh I cherry picked it from my other PR https://github.com/webmozart/assert/pull/193 which @BackEndTea probably hasn't merged due to mistake.
It's not that I wanted to smuggle it here though :-D But to avoid doing the same twice.